bftengine/src/preprocessor/messages/ClientPreProcessRequestMsg.cpp

```diff
@@ -23,15 +23,22 @@ ClientPreProcessRequestMsg::ClientPreProcessRequestMsg(NodeIdType sender,
                                                        uint32_t requestLength,
                                                        const char* request,
                                                        uint64_t reqTimeoutMilli,
-                                                       const std::string& cid)
-    : ClientRequestMsg(sender, PRE_PROCESS_REQ, reqSeqNum, requestLength, request, reqTimeoutMilli, cid) {
+                                                       const std::string& cid,
+                                                       const std::string& span_context)
```

Comment:
The addition of a new parameter should change both - message header and body. I miss this change here.

Reply:
Discussed offline